### ISSUE TITLE
[FEAT]: Addition of Logging

### ISSUE THREAD

**Issue by AbhijithGanesh** [ComputerSocietyVITC/Sahaay#42]
Logging methods can be implemented with Uvicorn and it has been loaded in this (repository)[https://github.com/AbhijithGanesh/Sahaay/blob/master/Core/Core/logging_conf.py]. @AbhijithGanesh has loaded the configuration for Uvicorn using custom logging.

Now, to configure the logs, we require logging methods to be implemented, It'd be required in the following places:
- [] (asgi.py)[https://github.com/ComputerSocietyVITC/Sahaay/blob/main/Core/Core/asgi.py]
- [] (auth.py)[https://github.com/ComputerSocietyVITC/Sahaay/blob/main/Core/Core/auth.py]
- [] Every Router and Endpoint which can be found in /docs

Basically, we need statements to be logged after every return statement and raise error statement,
An example would be:
```py
  if not user:
        logger.error("Authentication Error")
        raise HTTPException(status_code=400, detail="Incorrect username or password")

```
There are three logger methods:
 - info
 - error
 - debug

We need `logger.<methods>` established on every return/raise.
